doc/TUTORIAL_SCRIPT.md

```diff
@@ -0,0 +1,103 @@
+# Script
+
+```shell script
+docker run -it elektra/elektra                                     
+
+# Let's map the content of a hello.js file to Elektra's configuration in /hello
+sudo kdb mount hello.js /hello json
+
+# Set a value and retrieve it
+kdb set user/hello/hello/world "Hello World"
+kdb get user/hello/hello/world
+
+# You can also see where the file is located from Elektra
+kdb file user/hello/hello/world
+
+# You can edit the file and see what Elektra wrote
+vi /home/elektra/.config/hello.js
+>> Edit file to say "Hello World from File!"
+
+# Getting the value again ... 
+kdb get user/hello/hello/world
+
+# You can also use Elektra's editor which additionally validates the file content
+kdb editor user/hello ini
+>> Edit file to say "Hello World from INI via kdb editor!"
+kdb get user/hello/hello/world
+
+# Elektra can export to various file formats
+kdb export user/hello
+kdb export user/hello yaml                                      
+
+# You can search for plugins too
+kdb plugin-list
+kdb export user/hello tcl
+
+# Let's clean up our environment before we proceed
+kdb rm user/hello/hello/world
+
+# Removing the only entry also removed the whole file
+cat /home/elektra/.config/hello.ini
+
+# Remove the mountpoint too
+sudo kdb umount /hello
+
+# Let's see how configuration validation works
+# Validation works via metadata assigned to keys
+# You can find all metadata by searching through METADATA.ini which is mounted like this
+sudo kdb mount-info
+
+# Now the file is mounted under system/info/elektra/metadata/#0
+kdb mount
+
+# Let's search for an integer range validation
+kdb find system/info/elektra/metadata/#0/.*/range
+
+# So check/range can be used
+# Write a specification file with it
+vi specification.ini                                      
+>> []
+>> mountpoint = specification.ini
+>> [timeout]
+>> check/range = 0-100
+
+# Let's integrate the specification into the backend 
+# The 'ni' plugin can be used to parse INI files into Elektra
+sudo kdb mount /home/elektra/specification.ini spec/validation ni
+
+# Let's active the configuration validation
+sudo kdb spec-mount /validation
+
+# Now let's see how it works
+kdb set /validation/timeout 500
+
+# Alright, but what if you really want a value of 500?
+kdb editor spec/validation ni
+> edit range to 1000
+
+kdb set /validation/timeout 500
+
```

Comment:
Specification still not extended?

Reply:
I think that users will get the concept enough with one example and also when I edit the example as shown with the range editing. It is better to keep the tutorial a bit more simpler since it already is extending into a blockbuster movie length tutorial ...